### ISSUE TITLE
Add a personal profile section.

### ISSUE THREAD

**Issue by rajat2502** [rajat2502/GitHub-Battle#2]
A personalized section where users can see their contribution graphs and other stats will look good in this project.

**Comment by dsaharia** [comment]
I want to work on this issue for KWoC 2019.
For login I have to use Oauth right? then display information regarding stars, followers etc?

**Comment by rajat2502** [comment]
@dsaharia don't include login signup for now, just add a profile link to the navbar and in that component add a input box to enter your profile and then use the GitHub's API to show stats.

**Comment by rajat2502** [comment]
For reference about what to show in that component you can have a look at https://github.com/rajat2502/GitHub-Finder this app. Keep showing your progress in small steps so that I can approve and you can work further.

**Comment by dsaharia** [comment]
Yes, Sure. I'll keep you posted by Email?

**Comment by rajat2502** [comment]
Sure!!! but you can also join the Gitter channel from the KWoC website.

**Comment by rajat2502** [comment]
Are  you working?

**Comment by dsaharia** [comment]
Yes started today. As you have heard internet is cut in many places in Assam now

Dhiraj Saharia

________________________________

**Comment by rajat2502** [comment]
Ok take your time.

**Comment by dsaharia** [comment]
Internet is back here. I am working on it...
I will create a Profile section and then ask for the username of the user right then show him his stats. Right?

Regards,
Dhiraj Saharia,
B.Tech, CSE (3rd Year),
Tezpur University,
Napaam, Assam - 784028

On 15-Dec-2019, at 7:02 AM, Rajat Verma <notifications@github.com<mailto:notifications@github.com>> wrote:


Ok take your time.

**Comment by rajat2502** [comment]
@dsaharia Yeah that's cool.
For showing, loading add a loader.
Use the same input button as used in the battle component.
Add the profile route to the main navbar

**Comment by dsaharia** [comment]
I added the route and currently trying to display the results. I have pushed code.. Should I do a PR now ? for my evaluations?


On 17-Dec-2019, at 9:10 PM, Rajat Verma <notifications@github.com<mailto:notifications@github.com>> wrote:


@dsaharia<https://github.com/dsaharia> Yeah that's cool.
For showing, loading add a loader.
Use the same input button as used in the battle component.
Add the profile route to the main navbar

**Comment by rajat2502** [comment]
Just show me that by hosting.

**Comment by dsaharia** [comment]
hosting using netlify and my account?


On 18-Dec-2019, at 8:08 AM, Rajat Verma <notifications@github.com<mailto:notifications@github.com>> wrote:


Just show me that by hosting.

**Comment by dsaharia** [comment]
I have some doubts.
1. The `RepoCard` component takes `data.owner.avatar_url`. Why is there owner in between? When I try to use this component for profile section it gives error because there is no `owner` attribute in the data. What should I do?

**Comment by rajat2502** [comment]
Yeah, you can use netlify to host.
Well about the owner thing I am not sure, maybe there are some changes in GitHub's API
If it's working without owner use it that way.

**Comment by dsaharia** [comment]
It is working without using the `owner`. Should I host or send you screenshots of it? Because hosting will take time, and it is already hosted right?

**Comment by rajat2502** [comment]
Host your version of website, it's easy try and read about Netlify hosting.

**Comment by dsaharia** [comment]
I did some changes and added a profile section. It needs UI improvements, working on it. Also hosted it
LINK : https://fervent-lamarr-9bf0e5.netlify.com/
How is it?

**Comment by rajat2502** [comment]
Well many UI improvements are needed
![image](https://user-images.githubusercontent.com/42200276/71302612-415e6b80-23d3-11ea-8d62-98071bfb5517.png)
Also, none of the icons are loading. Please see to it.

**Comment by dsaharia** [comment]
It is loading in my pc. I have tested it.
Please see
![Screenshot 2019-12-21 at 6 46 37 PM](https://user-images.githubusercontent.com/29610517/71308411-439be680-2422-11ea-884c-711c23644368.png)

**Comment by rajat2502** [comment]
Not loading!!!

**Comment by dsaharia** [comment]
It should load. I have tested it in chrome as well as Firefox. Both loading fine.

**Comment by Neilblaze** [comment]
> It should load. I have tested it in chrome as well as Firefox. Both loading fine.

@dsaharia please check issue #11

**Comment by dsaharia** [comment]
There might be a issue. But I checked in Safari, Chrome and firefox. All the emoji's were loading fine.

**Comment by dsaharia** [comment]
How can I check, any clues?

**Comment by rajat2502** [comment]
Okay try to use any emoji service like fontawesome.

**Comment by dsaharia** [comment]
Done. Integrated FontAwesome for profile badges.
link: https://fervent-lamarr-9bf0e5.netlify.com
Should I open a PR now?

**Comment by rajat2502** [comment]
Well the profile component needs some better design (the design should match with the design of the rest of the page.) 
Also, this component should also include some charts and analysis with data. If the work is more I can divide into more than one person. 
For now, you can just work on the styling of the component.
Also, if possible try SVG for icons as font-awesome is a heavy library.

**Comment by dsaharia** [comment]
Yeah I am currently working on the styling and 1 more person would be good to do the charts and analysis, so that we would discuss it and then do it.

**Comment by rajat2502** [comment]
Okay you work on the styling for now.

**Comment by dsaharia** [comment]
Link: https://fervent-lamarr-9bf0e5.netlify.com
1. I have added a CSS file to match the profile section to other parts of the site.
2. Removed the fontawesome library.
3. Added SVG icons for the profile section. Working fine now. Also compatible with dark mode.
Please check.
Also I need help with the charts and other analysis.

**Comment by rajat2502** [comment]
Please decrease the width of the container that contains the total info.
